fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/util/PriorityMessageService.java

```diff
@@ -73,6 +73,8 @@ public static List<PriorityMessage> getPriorityMessages(final MavenSession sessi
             }
         } catch (JsonProcessingException e) {
             logger.error("Unable to get priority messages", e);
+        } catch (NullPointerException e) {
+            logger.error("No active Maven session available!", e);
```

Comment:
I: I more or less defaulted to treating this case as a no-op, as it's most likely to be met in a third party unit testing situation where a no-op makes sense.  With that said, I'm open to discussion of the best path to take here.